### PR TITLE
Make a delete request 

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #35

### CODE REVIEW

File: src/controllers/places/delete_place.js
Comment (m4v15) [inline src/controllers/places/delete_place.js]: sorry, but lets still send a 404 here (like line 15)

**Review by m4v15 — CHANGES_REQUESTED** [review]
Just that error needs to be change - I might hold off and wait to merge this until #34 is merged because you've included the checkOptions file here (by accident I'm guessing)

File: src/controllers/places/delete_place.js
Comment (jwld) [reply src/controllers/places/delete_place.js]: Should be a 500 yo

File: src/text.js
Comment (jwld) [inline src/text.js]: typo - missing the final e

**Review by jwld — CHANGES_REQUESTED** [review]
Typo needs fixing

**Pull request merged**
